benchmarks/dynamo/common.py

```diff
@@ -1529,10 +1558,10 @@ def parse_args():
 
 def main(runner, original_dir=None):
     args = parse_args()
-    return maybe_fresh_cache(run, args.cold_start_latency and args.only)(
-        runner, args, original_dir
-    )
-
+    with maybe_init_distributed(args.ddp and args.only):
```

Comment:
is `args.only` to avoid any distributed state issues?

Reply:
no, its a pattern for this suite.  `--only` will be present for any process that is actually going to run a benchmark.
1) you ran with `--only` yourself: it literally starts the benchmark.  (and we want to init ddp stuff)
2) you are sweeping over many models: the outer script (invoked without only) will subprocess once per model, and add the `--only {name}` arg to the subprocess